public/api.json

```diff
@@ -1 +1 @@
-[{"browser":"Chrome","os":"","versions":["112","113"],"device":"desktop"},{"browser":"Edge","os":"","versions":["112","113"],"device":"desktop"},{"browser":"Firefox","os":"","versions":["112","113"],"device":"desktop"},{"browser":"Safari","os":"","versions":[15,16],"device":"desktop"},{"browser":"Chrome","os":"Android","versions":["113"],"device":"mobile"},{"browser":"Firefox","os":"Android","versions":["113"],"device":"mobile"},{"browser":"Safari","os":"iOS","versions":[15,16],"device":"mobile"}]
\ No newline at end of file
+[{"browser":"Chrome","os":"","versions":["113","114"],"device":"desktop"},{"browser":"Edge","os":"","versions":["113","114"],"device":"desktop"},{"browser":"Firefox","os":"","versions":["113","114"],"device":"desktop"},{"browser":"Safari","os":"","versions":[15,16],"device":"desktop"},{"browser":"Chrome","os":"Android","versions":["114"],"device":"mobile"},{"browser":"Firefox","os":"Android","versions":["113"],"device":"mobile"},{"browser":"Safari","os":"iOS","versions":[15,16],"device":"mobile"}]
```

Comment:
According to https://www.mozilla.org/en-US/firefox/android/114.0/releasenotes/ Firefox Android 114 was released this month.